website/docs/docs/quickstarts/dbt-cloud/bigquery-qs.md

```diff
@@ -0,0 +1,186 @@
+    ```
+
+    Then, check for results from the queries. For example: 
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/bigquery/query-results.png" title="Bigquery Query Results" />
+    </div>
+2. Create [new BigQuery datasets](https://cloud.google.com/bigquery/docs/datasets#create-dataset). Datasets in BigQuery are equivalent to schemas in a traditional database. On the **Create dataset** page:
+    - **Dataset ID** &mdash; Enter a name that fits the purpose. This name is used like schema in fully qualified references to your database objects such as database.schema.table. As an example for this guide, create one for jaffle_shop and another one for stripe afterward.
+    - **Data location** &mdash; Leave it blank (the default). It determines the GCP location of where your data is stored. The current default location is the US multi-region. All tables within this dataset will share this location.
+    - **Enable table expiration** &mdash; Leave it unselected (the default). The default for the billing table expiration is 60 days. Because billing isn’t enabled for this project, GCP defaults to deprecating tables.
+    - **Google-managed encryption key** &mdash; Allow Google to manage encryption (the default).
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/bigquery/create-dataset-id.png" title="Bigquery Create Dataset ID" />
+    </div>
+3. After you create the `jaffle_shop` dataset, create one for `stripe` with all the same values except for **Dataset ID**.
+
+## Generate BigQuery credentials {#generate-bigquery-credentials}
+In order to let dbt connect to your warehouse, you'll need to generate a keyfile. This is analogous to using a database username and password with most other <Term id="data-warehouse">data warehouses</Term>.
+
+1. Start the [GCP credentials wizard](https://console.cloud.google.com/apis/credentials/wizard). Make sure your new project is selected in the header. If you do not see your account or project, click your profile picture to the right and verify you are using the correct email account. For **Credential Type**: 
+    - From the **Select an API** dropdown, choose **BigQuery API**
+    - Select **Application data** for the type of data you will be accessing
+    - Select **No, I’m not using them** and click **Next**.
+    - Click **Continue** in the popup to create a new service account.
+2. [Create a service account](https://developers.google.com/workspace/guides/create-credentials#service-account). As an example for this guide, you can:
+    - Enter `dbt-user` as the **Service account name**
+    - From the **Select a role** dropdown, choose **BigQuery Admin** 
+    - Leave the **Grant users access to this service account** fields blank
+3. [Create a service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating). When downloading the JSON file, make sure to use a filename you can easily remember. For example, `dbt-user-creds.json`. 
```

Comment:
Same comment as above about linking to the docs by saying, "For more information, refer to xyz in the Google BigQuery Documentation"